src/components/index/demo-display.tsx

```diff
@@ -24,8 +25,20 @@ const DemoDisplay = ({demo, descript}: DemoDisplayProps ) => {
         }
     }, [demo, descript]);
 
+    let additonalStyle:string = '';
+    
+    if (currentTab === 0) {
```

Comment:
Add a comment explaining what this does (i.e., changes the grey border around the demo-display when the top or bottom button is selected so that where the button connects there isn't a rounded corner)